### PR TITLE
[stable/external-dns] Fix README to reflect latest cloudflare parameter changes

### PR DESCRIPTION
**Pull request by thecodeassassin**
* Removed redundant cloudflare.apiKey from the configuration table
* fixed change to camelCase format recently introduced

### CODE REVIEW

**Review by timstoop — APPROVED** [review]
Small change of what was already present in the actual deployment.

**Review by timstoop — CHANGES_REQUESTED** [review]
As charts are immutable, every change needs to bump the version in Chart.yaml.

File: stable/external-dns/README.md [hunk 1]
```diff
@@ -27,9 +27,8 @@ The following tables lists the configurable parameters of the external-dns chart
...
 | `aws.zone_type`           | Filter for zones of this type (optional, options: public, private).                                                        | `""`                                               |
-| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                 | `""`                                        |
-| `cloudflare.email`        | `CF_API_EMAIL` to set in the environment (optional).                                                                 | `""`                                        |
-| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                         | `""`                                               |
+| `cloudflare.email`        | `CF_API_EMAIL` to set in the environment (optional).                                                                       | `""`                                               |
```
Comment (unguiculus) [inline stable/external-dns/README.md]: This is there twice `cloudflare.email`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by viglesiasce** [comment]
Already fixed in master

**Pull request closed without merging**